.ci/pytorch/test.sh

```diff
@@ -521,6 +521,51 @@ test_inductor_torchbench_smoketest_perf() {
   done
 }
 
+test_inductor_torchbench_cpu_smoketest_perf(){
+  TEST_REPORTS_DIR=$(pwd)/test/test-reports
+  mkdir -p "$TEST_REPORTS_DIR"
+
+  #set jemalloc
+  #JEMALLOC_LIB="$(dirname "$(which python)")/../lib/libjemalloc.so"
+  JEMALLOC_LIB="/usr/lib/x86_64-linux-gnu/libjemalloc.so.2"
+  IOMP_LIB="$(dirname "$(which python)")/../lib/libiomp5.so"
```

Comment:
ditto: how do we make sure libiomp is installed?

Reply:
The base docker image build need install `mkl` conda package, and it depends on `intel-openmp` conda package. Refer: https://github.com/pytorch/pytorch/blob/main/.ci/docker/common/install_conda.sh#L54